teku/src/main/java/tech/pegasys/teku/cli/subcommand/debug/DebugCommand.java

```diff
@@ -24,15 +33,54 @@
     showDefaultValues = true,
     abbreviateSynopsis = true,
     mixinStandardHelpOptions = true,
+    addMethodSubcommands = true,
     versionProvider = PicoCliVersionProvider.class,
     synopsisHeading = "%n",
     descriptionHeading = "%nDescription:%n%n",
     optionListHeading = "%nOptions:%n",
     footerHeading = "%n",
     footer = "Teku is licensed under the Apache License 2.0")
 public class DebugCommand implements Runnable {
+  @Spec private CommandSpec commandSpec;
+
   @Override
   public void run() {
     CommandLine.usage(this, System.out);
   }
+
+  @Command(
+      name = "generate-autocomplete",
+      description = "Generate a bash/zsh autocomplete file",
+      subcommands = {DebugDbCommand.class},
+      showDefaultValues = true,
+      abbreviateSynopsis = true,
+      mixinStandardHelpOptions = true,
+      versionProvider = PicoCliVersionProvider.class,
+      synopsisHeading = "%n",
+      descriptionHeading = "%nDescription:%n%n",
+      optionListHeading = "%nOptions:%n",
+      footerHeading = "%n",
+      footer = "Teku is licensed under the Apache License 2.0")
+  public int autocomplete(
+      @Option(
+              names = {"--output", "-o"},
+              description = "File to output to, default is System.out")
+          final File output) {
+    try (final PrintStream out = outputStream(output)) {
```

Comment:
I believe we are exiting after executing this command, however I would avoid closing `System.out` though

Reply:
Sorry, I missed this comment before merging.  I must learn not to use the mobile GitHub app...

Fixed in https://github.com/PegaSysEng/teku/pull/2237